sklearn/tests/test_base.py

```diff
@@ -436,6 +436,7 @@ def __init__(self, attribute_pickled=5):
 
     def __getstate__(self):
         data = self.__dict__.copy()
+        data["_sklearn_version"] = sklearn.__version__
```

Comment:
Not in this test case, [here](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/tests/test_base.py#L412) we test estimators outside of sklearn.